lib/validator.js

```diff
@@ -338,7 +338,7 @@ Specberus.prototype.getFeedbackDueDate = function() {
 
 Specberus.prototype.getDelivererIDs = function () {
     const REGEX_DELIVERER_URL = /^((https?:)?\/\/)?(www\.)?w3\.org\/2004\/01\/pp-impl\/\d+\/status(#.*)?$/i
-    ,   REGEX_DELIVERER_TEXT = /^public\s+list\s+of\s+any\s+patent\s+disclosures(\s+\(.+\))?$/i
+    ,   REGEX_DELIVERER_TEXT = /^(charter|public\s+list\s+of\s+any\s+patent\s+disclosures(\s+\(.+\))?)$/i
```

Comment:
For IG notes, we don't have the regular patent policy boilerplate. So there's no "public list of any patent disclosures" link we can extract to get the deliverer ID.
However, the editors need to provide the charter disclosure link for that type of documents and from the [respec doc](https://github.com/w3c/respec/wiki/charterDisclosureURI), it does use the same link syntax. The resulting snapshot (e.g. [pwp](https://w3c.github.io/dpub-pwp/)) has the link on the word 'charter'.